images/prometheus-postgres-exporter/tests/main.tf

```diff
@@ -0,0 +1,49 @@
+terraform {
+  required_providers {
+    oci = { source = "chainguard-dev/oci" }
+  }
+}
+
+variable "digest" {
+  description = "The image digest to run tests over."
+}
+
+data "oci_exec_test" "version" {
+  digest = var.digest
+  script = "docker run --rm $IMAGE_NAME --version"
+}
+
+data "oci_string" "ref" { input = var.digest }
+
+resource "random_pet" "suffix" {}
+
+resource "helm_release" "bitnami" {
+  name             = "postgres-exporter-bitnami-${random_pet.suffix.id}"
+  repository       = "oci://registry-1.docker.io/bitnamicharts"
+  chart            = "postgresql"
+  namespace        = "postgres-exporter-bitnami-${random_pet.suffix.id}"
+  create_namespace = true
+
+  values = [jsonencode({
+    auth = {
+      postgresPassword = "${random_pet.suffix.id}"
+      username         = "${random_pet.suffix.id}"
+      password         = "${random_pet.suffix.id}"
+      database         = "${random_pet.suffix.id}"
+    }
```

Comment:
Are these required to get the helm test to pass? I was able to pass it (with the older version of `postgres-exporter` so, grain of salt here) with just

```
values = [jsonencode({
    metrics = {
      image = {
        registry   = data.oci_string.ref.registry
        repository = data.oci_string.ref.repo
        tag        = data.oci_string.ref.pseudo_tag
      }
    }
  })]
```

Reply:
Yeah, the `metrics` container has to be `enabled` to spin up the postgres exporter. When I toggled that `enabled` flag it wanted me to specify a database and some creds so the exporter knows where to listen

Reply:
https://github.com/chainguard-images/images/pull/1647/files#diff-6c21e78c6356024de6b7ecfade2f1dc00fe68eb54c1ba4b3a2a271346fb748e1R35